src/app/project/ontology/ontology.component.scss

```diff
@@ -45,15 +44,29 @@ $width: 340px;
     height: $sub-header-height;
   }
 
+  .ontology-editor-sidenav {
+    width: 160px;
+
+    button {
+      width: 100%;
+      text-align: left;
+    }
+  }
+
   .ontology-editor-canvas {
     min-height: calc(100vh - #{$sub-header-height} - #{$header-height} - #{$tab-bar-margin} - 100px);
+    // background: $primary_200;
```

Comment:
when there are unused css classes, we should clean them up or add a comment (remark valid for all the scss files)

Reply:
Some of the css classes are not defined in the scss file, that's true. But they are needed for the tests.

Or are you talking about the commented css property here? I can clean them up...

Reply:
Done in ac7c3cf